docs/gallery.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# Generates images for the gallery
+
+import os
+import argparse
+import numpy as np
+import os.path as path
+import matplotlib.pyplot as plt
+
+from yellowbrick.datasets import load_occupancy, load_credit, load_concrete
+
+from yellowbrick.features import RFECV, JointPlot
+from yellowbrick.features import RadViz, Rank1D, Rank2D, ParallelCoordinates
+from yellowbrick.features import PCADecomposition, Manifold, FeatureImportances
+
+from yellowbrick.contrib.scatter import ScatterVisualizer
+
+from sklearn.ensemble import RandomForestClassifier
+
+
+GALLERY = path.join(path.dirname(__file__), "images", "gallery")
+
+
+##########################################################################
+## Helper Methods
+##########################################################################
+
+def newfig():
+    """
+    Helper function to create an axes object of the gallery dimensions.
+    """
+    # NOTE: this figsize generates a better thumbnail
+    _, ax = plt.subplots(figsize=(8,4))
```

Comment:
Setting this figsize makes the thumbnails a lot better looking.

Reply:
This is the new features gallery:

<img width="655" alt="screenshot 2019-02-07 22 21 51" src="https://user-images.githubusercontent.com/745966/52457279-0343ba80-2b27-11e9-8b35-fea53d52e5ea.png">


This is the old one:

<img width="636" alt="screenshot 2019-02-07 22 21 43" src="https://user-images.githubusercontent.com/745966/52457281-063eab00-2b27-11e9-8ee9-6d14a8fffbca.png">

Reply:
You are right, the new ones do look better